frontend/src/Components/Map/Inputs/Hamburger.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import '../../../css/Sidebar.css';
+import { getAllSurveyData } from '../../../queries/conditions';
+import { SurveyList } from '../../../../../backend/src/models';
+
+/**
+ * Defines the properties for the Hamburger component, which include the state of the sidebar
+ * and a function to toggle its visibility.
+ *
+ * @property {boolean} isOpen Indicates if the sidebar is currently open.
+ * @property {Function} toggle A function to open or close the sidebar.
+ *
+ * @author Lyons
+ */
+interface HamburgerProps {
+  isOpen: boolean; // Whether the sidebar is open or not
+  toggle: () => void;
+  //surveys: Survey[];
+}
```

Comment:
```suggestion
interface HamburgerProps {
  /** Indicates if the sidebar is currently open. */
  isOpen: boolean;
  /** A function to open or close the sidebar. */
  toggle: () => void;
}
```
I pretty sure the [auto documentation builder](https://dtu-se-group-d.github.io/LiRA-Viz/frontend/) wouldn't understand it

Reply:
@Seb-sti1 Awesome! Changes have been made. Thanks!